judo-ui-react/src/main/resources/actor/src/components/MdiIcon.tsx.hbs

```diff
@@ -26,9 +27,11 @@ export const MdiIcon = (props: MdiIconProps) => {
     height: '1rem',
     lineHeight: '1rem',
   };
-  const className = `mdi mdi-${mappedPath.replace(/_/g, '-').replace(/ /g, '')} ${rotate ? `mdi-rotate-${rotate}` : ''} ${flip ? `mdi-flip-${flip}` : ''} ${spin ? 'mdi-spin' : ''}`.trim();
+  const classes = `mdi mdi-${mappedPath.replace(/_/g, '-').replace(/ /g, '')} ${
```

Comment:
Not that it matter that much, but we should follow the ternary pattern, because we are closing with a `.trim()`, and not with a potentially empty concat.

Reply:
[8df3fb6](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/159/commits/8df3fb6c835aa8c77f152ad93256f85c23476867)